drivers/pinctrl/pinctrl_stm32.c

```diff
@@ -0,0 +1,588 @@
+/*
+ * Copyright (c) 2017 Bobby Noelte
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Keep this first */
+#include "pinctrl_stm32.h"
+
+#include <errno.h>
+#include <string.h>
+#include <zephyr/types.h>
+#include <misc/__assert.h>
+#include <misc/util.h>
+#include <kernel.h>
+#include <device.h>
+#include <soc.h>
+#include <gpio.h>
+#include <pinctrl.h>
+
+#include <clock_control/stm32_clock_control.h>
+
+#include <chaos/preprocessor/array/elem.h>
+#include <chaos/preprocessor/cat.h>
+#include <chaos/preprocessor/control/when.h>
+#include <chaos/preprocessor/repetition/repeat_from_to.h>
+
+
```

Comment:
useless empty line

Reply:
Sorry, currently there are a lot of these things. Wanted to get that whole thing out to get feedback. Will look into it. Thank you for reviewing.